cli/tests/upgrade-tests/main_test.go

```diff
@@ -0,0 +1,73 @@
+package upgrade_tests
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"os/signal"
+	"testing"
+	"time"
+
+	test_utils "github.com/timescale/tobs/cli/tests/test-utils"
+)
+
+var PATH_TO_TOBS = "./../../bin/tobs"
+var PATH_TO_CHART = "./../../../chart/"
+var NAMESPACE = "ns"
+var RELEASE_NAME = "gg"
+
+func TestMain(m *testing.M) {
+	// Signal handling
+	sigchan := make(chan os.Signal, 1)
+	done := make(chan bool, 1)
+	signal.Notify(sigchan, os.Interrupt)
+	go func() {
+		<-sigchan
+		done <- true
+		os.Exit(1)
+	}()
+
+	installTobsRecentRelease()
+
+	upgradeTobsLatest()
+
+	fmt.Println("Successfully upgraded tobs to latest version")
+
+	code := m.Run()
+
+	// upgrade takes some time for pods to get into running state
+	time.Sleep(2 * time.Minute)
+
+	err := test_utils.CheckPodsRunning(NAMESPACE)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	os.Exit(code)
+}
+
+func installTobsRecentRelease() {
+	// Note: The below tobs cmd only deploys TimescaleDB as the test values.yaml is configured
```

Comment:
why are we doing only timescaleDB here?

Reply:
The comment line is outdated. Will update the somment line.